package.json

```diff
@@ -22,9 +22,11 @@
     "test:all": "ember try:each"
   },
   "dependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-decorators": "^7.0.0",
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "ember-cli-babel": "^6.6.0",
+    "ember-cli-babel": "^7.1.0",
```

Comment:
I'm not actually sure that it's necessary to have `ember-cli-babel` in the `dependencies` at all, since this addon has no `addon/` directory. It might make more sense to move it to `devDependencies` instead. Thoughts?

Reply:
I _think_ that makes sense, would probably be a good idea to remove it

cc @rwjblue

Reply:
Agreed, makes sense to move to devDependencies to me...